composer.json

```diff
@@ -88,7 +88,6 @@
         "drupal/migrate_tools": "8.2.0-beta1",
         "drupal/oauth": "8.1.x-dev",
         "drupal/page_manager": "8.1.0-alpha23",
-        "drupal/panels": "8.3.x-dev",
```

Comment:
I think typically we had been running panels on dev because @mortenson had a lot of patches for it due to the IPE being heavily worked on.